### PR TITLE
Fix identity prefixes in openconfig-aaa.yang when statements

### LINKED ISSUE #244: openconfig-aaa.yang uses grouping has incorrect when statement prefixes
In openconfig-aaa.yang, the following uses grouping have incorrect when statement prefix
```
       uses aaa-tacacs-server-top {
          when "../../config/type = 'oc-aaa-types:TACACS'";
        }

        uses aaa-radius-server-top {
          when "../../config/type = 'oc-aaa-types:RADIUS'";
        }
```

Identity TACACS is defined in openconfig-aaa-tacacs.yang, a submodule of openconfig-aaa.yang
```
  identity TACACS {
    base oc-aaa-types:AAA_SERVER_TYPE;
    description
      "Terminal Access Controller Access Control System (TACACS+)
      AAA server";
    reference
      "The TACACS+ Protocol (draft-ietf-opsawg-tacacs-05)
      RFC 1492 - An Access Control Protocol, Sometimes Called
      TACACS";
  }
```

Identity RADIUS is defined in openconfig-aaa-radius.yang, a submodule of openconfig-aaa.yang
```
  identity RADIUS {
    base oc-aaa-types:AAA_SERVER_TYPE;
    description
      "Remote Authentication Dial In User Service (RADIUS) AAA
      server";
    reference
      "RFC 2865 - Remote Authentication Dial In User Service
      (RADIUS)";
  }
```

Both identities are defined in submodules of openconfig-aaa.yang, then according to https://tools.ietf.org/html/rfc7950#section-6.2.1,
   o  All identity names defined in a module and its submodules share
      the same identity identifier namespace.

The when statement prefix should be `oc-aaa:TACACS` and `oc-aaa:RADIUS` instead of `oc-aaa-types:TACACS` and `oc-aaa-types:RADIUS`, respectively.

### PR DESCRIPTION
**Pull request by LeonGWang**
* (M) release/models/system/openconfig-aaa.yang

  This Pull Request closes #244

### CODE REVIEW

**Comment by LeonGWang** [comment]
I signed it!

**Comment by LeonGWang** [comment]
I signed it!

File: release/models/system/openconfig-aaa.yang [hunk 1]
```diff
@@ -32,7 +32,14 @@ module openconfig-aaa {
     Portions of this model reuse data definitions or structure from
     RFC 7317 - A YANG Data Model for System Management";
 
-  oc-ext:openconfig-version "0.4.1";
+  oc-ext:openconfig-version "0.4.2";
```
Comment (exa-arrcus) [inline release/models/system/openconfig-aaa.yang]: Believe any adjustments to version/revision will be done at time of release to batch fixes/changes so unlikely unnecessary here

Comment (LeonGWang) [reply release/models/system/openconfig-aaa.yang]: Thanks. I have removed revision changes.

**Comment by exa-arrcus** [comment]
@aashaikh @robshakir - could we get someone assigned for approval here?  There are a number of these bugs scattered throughout that make implementors have to hand-edit upstream models in various cases

**Comment by aashaikh** [comment]
I have a corresponding change pending for this particular issue and will get the other bugfixes addressed as well.

**Comment by LeonGWang** [comment]
This PR is incorporated in the commit 54c3ec866b0302998ab8f9fcf70e6dcd0cc79665

**Comment by exa-arrcus** [comment]
Note that it appears that the committed when statements are now incorrect.  It should rather be

```
@@ -272,11 +274,11 @@ module openconfig-aaa {
         }
 
         uses aaa-tacacs-server-top {
-          when "../../../config/type = 'oc-aaa:TACACS'";
+          when "../../config/type = 'oc-aaa:TACACS'";
         }
 
         uses aaa-radius-server-top {
-          when "../../../config/type = 'oc-aaa:RADIUS'";
+          when "../../config/type = 'oc-aaa:RADIUS'";
         }
       }
     }
```

**Pull request closed without merging**